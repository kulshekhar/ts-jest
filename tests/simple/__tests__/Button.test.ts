import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Button from './../Button.tsx';

it('renders correctly', () => {
  console.log('Button',Button);
  const tree = renderer.create(<Button>hi!</Button>).toJSON();
  expect(tree).toMatchSnapshot()
})
