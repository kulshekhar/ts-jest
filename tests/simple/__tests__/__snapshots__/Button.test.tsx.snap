exports[`test renders correctly 1`] = `
<div>
  hi!
</div>
`;
